controller/base.php

```diff
@@ -0,0 +1,70 @@
+<?php
+/**
+ *
+ * This file is part of the phpBB Forum Software package.
+ *
+ * @author Callum Macrae (callumacrae) <callum@lynxphp.com>
+ * @copyright (c) phpBB Limited <https://www.phpbb.com>
+ * @license GNU General Public License, version 2 (GPL-2.0)
+ *
+ * For full copyright and license information, please see
+ * the docs/CREDITS.txt file.
+ *
+ */
+
+namespace phpbb\ideas\controller;
+
+// @todo: refactor out
```

Comment:
Will be refactored out in a separate PR after this one is merged, adding a new ACP module to choose a forum and user.